pkg/sandbox/start.go

```diff
@@ -287,21 +395,75 @@ func StartCluster(ctx context.Context, args []string, sandboxConfig *sandboxCmdC
 	return nil
 }
 
+func confirmAndRemoveIfExists(fname string, userInput io.Reader) error {
+	if _, err := os.Stat(fname); os.IsNotExist(err) {
+		return nil
+	}
+	msg := fmt.Sprintf("This will overwrite the existing Flyte config file at [%s]. Do you want to continue?", fname)
+	if cmdUtil.AskForConfirmation(msg, userInput) {
+		if err := os.Remove(fname); err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("leaving %s in place, not removing", fname)
+	}
+	return nil
+}
+
+func DemoClusterInit(ctx context.Context, sandboxConfig *sandboxCmdConfig.Config, userInput io.Reader) error {
+	sandboxImagePrefix := "sha"
+
+	if err := confirmAndRemoveIfExists(docker.FlyteBinaryConfig, userInput); err != nil {
+		fmt.Printf("Leaving init file %s in place and skipping rest of initialization", docker.FlyteBinaryConfig)
+		return nil
+	}
+
+	if err := util.SetupFlyteDir(); err != nil {
+		return err
+	}
+
+	cli, err := docker.GetDockerClient()
+	if err != nil {
+		return err
+	}
+	ghRepo := github.GetGHRepoService()
+
+	// Determine and pull the image
+	sandboxImage := sandboxConfig.Image
+	if len(sandboxImage) == 0 {
+		image, _, err := github.GetFullyQualifiedImageName(sandboxImagePrefix, sandboxConfig.Version, demoImageName, sandboxConfig.Prerelease, ghRepo)
+		if err != nil {
+			return err
+		}
+		sandboxImage = image
+	}
+	fmt.Printf("%v Fetching image %s\n", emoji.Whale, sandboxImage)
+	err = docker.PullDockerImage(ctx, cli, sandboxImage, docker.ImagePullPolicyIfNotPresent, docker.ImagePullOptions{})
+	if err != nil {
+		return err
+	}
```

Comment:
Sandbox Config has ImagePullPolicy and PullOptions. can this be used when calling this function instead of the default  docker.ImagePullPolicyIfNotPresent and docker.ImagePullOptions